src/routes/tasks.js

```diff
@@ -105,8 +106,12 @@ router.get("/:id", (request, response) => {
  *        type: string
  * */
 router.patch("/:id", (request, response) => {
-    Tasks.updateOne({ _id: request.params.id }, { $set: request.body })
-        .then((resp) => response.status(200).json(resp))
+    Tasks.findByIdAndUpdate(
+        { _id: request.params.id },
+        { $set: request.body },
+        { returnDocument: "after" }
```

Comment:
added `{ returnDocument: "after" }` which is required to return the document after being PATCHed